### ISSUE TITLE
Орфоварианты

### ISSUE THREAD

**Issue by grandsbor** [OpenCorpora/opencorpora#641]
(найдены @yourmary)
[Раз список](https://yadi.sk/d/VlzXiiQeh4oCe), [два список](https://yadi.sk/d/TfnsdbRBh4oDV)
Надо расставить связи.

сс @svbichineva

**Comment by svbichineva** [comment]
Вроде бы списки совпадают. Ты сам это сделаешь?

**Comment by grandsbor** [comment]
Не должны совпадать, вроде. Кто сделает - не знаю, кажется, можно даже вручную.

**Comment by gisly** [comment]
Добавила возможность обработки файлов из приложения скриптом. Есть вопросы:
1) в файле nonnames есть слово "РУКОПЛЛЕСКАНИЕ", имеющие уже помету "удалено"
2) приложила список имен, которые встречаются в базе несколько раз
[names_duplicates.txt](https://github.com/OpenCorpora/opencorpora/files/189121/names_duplicates.txt)

3) сейчас не обрабатываю дублирующиеся строки в файле (таких много в файле names).

**Comment by grandsbor** [comment]
1) значит, его можно игнорировать
2-3) значит, связи нужно делать со всеми

**Comment by gisly** [comment]
2-3) добавила возможность указать при выполнении скрипта process_links параметр several_lexemes, который ищет несколько лексем по заданным лемме+граммемам, а не только одну. Наверное, лучше считать, что по умолчанию такой ситуации не может быть? Или выкинуть параметр и считать, что дубликаты — это нормально?

**Comment by grandsbor** [comment]
Нет, дубликаты - это ненормально.
А можно включить туда режим типа "dry run", в котором ничего реально с базой не происходит, а только пишется в stdout (или stderr), что должно произойти?

**Comment by gisly** [comment]
да, добавила опцию -d — тогда только пишутся инсерты, а реально вставок не делается
